packages/transaction-controller/package.json

```diff
@@ -69,13 +69,12 @@
   },
   "devDependencies": {
     "@babel/runtime": "^7.23.9",
-    "@metamask/accounts-controller": "^19.0.0",
+    "@metamask/accounts-controller": "^20.0.0",
     "@metamask/approval-controller": "^7.1.1",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/eth-json-rpc-provider": "^4.1.6",
     "@metamask/ethjs-provider-http": "^0.3.0",
     "@metamask/gas-fee-controller": "^22.0.1",
-    "@metamask/keyring-api": "^10.1.0",
```

Comment:
Should we add this back?

Reply:
Yup, I think it was missing cause my `main` branch was not up-to-date at that time (or something like that).

It seems to have resolved itself while merging `main` back into this branch.